### PR TITLE
use liters for blood volume variable

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- Change the magic 100 of blood volume to 6 liters.
- also add macros / config to adjust certain other magic numbers
- change what is recorded as "lost some" or "a lot of" blood.

### CODE REVIEW

**Comment by commy2** [comment]
```
 +#define DEFAULT_BLOOD_VOLUME 6.0 // in liters
 +
 +#define HAS_LOST_SOME_BLOOD_THRESHOLD 0.100 // in liters
 +#define HAS_LOST_MUCH_BLOOD_THRESHOLD 0.500 // in liters
 +
 +#define BLOOD_VOLUME_UNCONSCIOUS 3.6 // in liters
 +#define BLOOD_VOLUME_DEAD 1.8 // in liters
 +#define BLOOD_VOLUME_CARDIAC_ARREST 1.2 // in liters
```

**Comment by commy2** [comment]
https://en.wikipedia.org/wiki/Bleeding#Blood_loss
```Class I Hemorrhage involves up to 15% of blood volume. There is typically no change in vital signs and fluid resuscitation is not usually necessary.
Class II Hemorrhage involves 15-30% of total blood volume. A patient is often tachycardic (rapid heart beat) with a narrowing of the difference between the systolic and diastolic blood pressures. The body attempts to compensate with peripheral vasoconstriction. Skin may start to look pale and be cool to the touch. The patient may exhibit slight changes in behavior. Volume resuscitation with crystalloids (Saline solution or Lactated Ringer's solution) is all that is typically required. Blood transfusion is not typically required.
Class III Hemorrhage involves loss of 30-40% of circulating blood volume. The patient's blood pressure drops, the heart rate increases, peripheral hypoperfusion (shock), such as capillary refill worsens, and the mental status worsens. Fluid resuscitation with crystalloid and blood transfusion are usually necessary.
Class IV Hemorrhage involves loss of >40% of circulating blood volume. The limit of the body's compensation is reached and aggressive resuscitation is required to prevent death.
```

85% - 5.1l
70% - 4.2l
60% - 3.6l

**Comment by commy2** [comment]
change ?

```
 #define BLOOD_VOLUME_UNCONSCIOUS 4.2
 #define BLOOD_VOLUME_DEAD 3.6
```
?

What about `BLOOD_VOLUME_CARDIAC_ARREST` ?

File: addons/medical/functions/fnc_handleUnitVitals.sqf [hunk 1]
```diff
@@ -23,27 +22,24 @@ if (_syncValues) then {
...
 
 _unit setVariable  [QGVAR(bloodVolume), _bloodVolume, _syncValues];
 
 TRACE_3("ACE_DEBUG",_bloodVolume,_syncValues,_unit);
 // Set variables for synchronizing information across the net
-if (_bloodVolume < 100) then {
-    if (_bloodVolume < 90) then {
-        TRACE_4("ACE_DEBUG",_bloodVolume,_unit getVariable QGVAR(hasLostBlood),_syncValues,_unit);
+if (_bloodVolume < DEFAULT_BLOOD_VOLUME - HAS_LOST_SOME_BLOOD_THRESHOLD) then {
+    if (_bloodVolume < DEFAULT_BLOOD_VOLUME - HAS_LOST_MUCH_BLOOD_THRESHOLD) then {
```
Comment (Glowbal) [inline addons/medical/functions/fnc_handleUnitVitals.sqf]: Bonus points:

We can simplify these if statements by using the `HAS_LOST_BLOOD_THRESHOLD` macros as thresholds, ie

```sqf
if (_bloodVolume < HAS_LOST_SOME_BLOOD_THRESHOLD) then {
..
}
```

Comment (commy2) [inline addons/medical/functions/fnc_handleUnitVitals.sqf]: I agree. Will do that.

**Pull request merged**